src/scripts/inclusion-bot.yaml

```diff
@@ -61,6 +61,11 @@ triggers:
       - unaware of
       - ignoring
       - not paying attention to
+    why: >
+      ":TRIGGER:" refers to inattention or ignorance. Blind, Deaf, and other
+      visually- or hearing-impaired people are not inattentive or ignorant.
```

Comment:
[very tiny nit] technically `visually impaired` (ends in ly) so you could get rid of that `-`